storage/innobase/pars/pars0opt.cc

```diff
@@ -553,7 +553,7 @@ opt_search_plan_for_table(
 	plan_t*		plan;
 	dict_index_t*	index;
 	ulint		n_fields;
-	ulint		best_last_op;
+	ulint		best_last_op= 0;
```

Comment:
Can you please also add a comment that says which compiler would emit a warning? If I remember correctly, until fairly recently (starting somewhere between GCC 7 and 9), many `-Wmaybe-uninitialized` were being emitted for no reason.

Reply:
The warning is generated from gcc-7.5.0, and the comment is added accordingly.